### ISSUE TITLE
compare property when rightvalue is null

### ISSUE THREAD

**Issue by jackletter** [NimaAra/Easy.Common#37]
reference code：
```sharp
//Variance.cs：
private bool VariesImpl()
        {
            var isSequence = Property.PropertyType.IsSequence(out SequenceType _);

            if (LeftValue is null) { return RightValue != null; }
            if (RightValue is null) { return false; }

            if (!isSequence) { return !Equals(LeftValue, RightValue); }

            var left = ((IEnumerable)LeftValue).Cast<object>();
            var right = ((IEnumerable)RightValue).Cast<object>();
            return !left.SequenceEqual(right);
        }
```
qusetion:

` if (RightValue is null) { return false; }` shouldbe ` if (RightValue is null) { return true; }`

becase, LeftValue is not null and RightValue  is null

**Comment by jackletter** [comment]
收到了 哈哈

**Comment by NimaAra** [comment]
Thank you for reporting this. It should be fixed as of [4.4.0](https://www.nuget.org/packages/Easy.Common/4.4.0)
